lint pkg/minikube/translate/translations.go

lint sometimes yells at this :

```
medya@~/workspace/minikube (master) $ make lint
./out/linters/golangci-lint-v1.21.0 run --timeout 4m --build-tags "integration container_image_ostree_stub containers_image_openpgp go_getter_nos3 go_getter_nogcs" --enable goimports,gocritic,golint,gocyclo,misspell,nakedret,stylecheck,unconvert,unparam,dogsled --exclude 'variable on range scope.*in function literal|ifElseChain' ./...
pkg/minikube/translate/translations.go:44:25: ST1005: error strings should not be capitalized (stylecheck)
                return nil, fmt.Errorf("Read %q: %v", name, err)
                                      ^
pkg/minikube/translate/translations.go:52:25: ST1005: error strings should not be capitalized (stylecheck)
                return nil, fmt.Errorf("Read %q: %v", name, err)
                                      ^
make: *** [lint] Error 1
```
I am not sure how to edit this file, since it is auto generated . @sharifelgamal do you know ?

--- a/Makefile
+++ b/Makefile
@@ -60,7 +60,8 @@ GOLINT_GOGC ?= 100
 GOLINT_OPTIONS = --timeout 4m \
 	  --build-tags "${MINIKUBE_INTEGRATION_BUILD_TAGS}" \
 	  --enable goimports,gocritic,golint,gocyclo,misspell,nakedret,stylecheck,unconvert,unparam,dogsled \
-	  --exclude 'variable on range scope.*in function literal|ifElseChain'
+	  --exclude 'variable on range scope.*in function literal|ifElseChain' \
+	  --skip-files "pkg/minikube/translate/translations.go|pkg/minikube/assets/assets.go"
 
 # Major version of gvisor image. Increment when there are breaking changes.
 GVISOR_IMAGE_VERSION ?= 2

